src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -5700,6 +5700,16 @@ public void updateSnitch(String epSnitchClassName, Boolean dynamic, Integer dyna
         updateTopology();
     }
 
+    public String getBatchlogEndpointStrategy()
+    {
+        return DatabaseDescriptor.getBatchlogEndpointStrategy().name();
+    }
+
+    public void setBatchlogEndpointStrategy(String batchlogEndpointStrategy)
+    {
+        DatabaseDescriptor.setBatchlogEndpointStrategy(Config.BatchlogEndpointStrategy.valueOf(batchlogEndpointStrategy));
```

Comment:
Consider some mechanism to deal with invalid `batchlogEndpointStrategy` values. For example, `ConsistencyLevel` has a `fromString` method.
In case of invalid values, it might also be nice to catch `IllegalArgumentException` and log a friendly message rather than a possible stacktrace. But I have not looked to see what the standard practice is for they type of thing in this class/area.